beehive-data-loader/data-loader.py

```diff
@@ -81,27 +114,39 @@ def unpack_messages_datagrams_sensorgrams(body):
             plugin_version = get_plugin_version(datagram)
             logging.exception('invalid message from node_id %s plugin %s %s with body %s',
                               node_id, plugin_id, plugin_version, body)
+            error_counter(node_id) #########
 
 
 csvout = csv.writer(sys.stdout)
 
 
+
 def get_plugin_version(datagram):
     return '{plugin_major_version}.{plugin_minor_version}.{plugin_patch_version}'.format(**datagram)
 
 
+
+flag = 0
+
 def message_handler(ch, method, properties, body):
     for message, datagram, sensorgram in unpack_messages_datagrams_sensorgrams(body):
         ts = datetime.datetime.fromtimestamp(sensorgram['timestamp'])
         node_id = message['sender_id']
+        global flag
+
+        if flag == 0:
+            node_id = '0000000000000002'
+            flag = 1
+        else :
+            flag = 0
```

Comment:
This part can be removed... I manually added a different node id to make dynamic panels for Grafana